### PR TITLE
fix: update navbar

### PR DESCRIPTION
**Pull request by janniks**
**tl;dr changes:**

- switch stacks-api item to references dropdown
- updates logo (now uses core-white and core-black color codes)
- changes navbar items positions
- fixes external link arrow icon across page

---

Changed the `Stacks API` menu item into a dropdown `References` — now includes all the references developers might need (even external Clarity stuff)
Screen Shot 2022-09-23 at 16 44 17

---

The current navbar feels a bit off to me, so I played around a bit. 
My main issue was the more textual logo clashing with the text-items in the navbar, so I tried rearranging some times.
Screen Shot 2022-09-23 at 16 08 16 _(`Doc` on `Doc` would be hard to get the different fonts to align or vibe with each other)_

Some rearrangements _(we can try more and adjust spacing as well)_:
Screen Shot 2022-09-23 at 15 49 The last is my favorite (in times where we don't have a dev survey we could replace the CTA with `Get Started` or something similar)

---

Additionally, I tightened up the letter spacing on the `: Docs` part of the logo:
Screen Shot 2022-09-23 at 15 29 42 Different fonts can have weird letter spacings for letter combinations and the three round letters didn't feels right with this amount of letter spacing.
@eugeniadigon thoughts?

---

Also, here's a little sneak peak (photoshopped) what orange could look like (not implemented yet) 😅
hiro-docs-replace-color-orange-2

### CODE REVIEW

**Comment by eugeniadigon** [comment]
Jannik...!! Thank you so much for working on this. The visually corrected logo looks great

I agree the last header option is the best (Logo --------------- Nav / GH / Theme / Search)
I would put the CTA at the right corner, after the search bar. Nonetheless, this button should be deactivated for now since the survey is finished (I'll double check before making the formal request).

And here's a design I made a while back for the landing page https://www.figma.com/file/ccBGDBwdBQcHWvKBHEZw5M/Hiro-Docs?node-id=0%3A1
Probably not all of this can be implemented on this platform, but I updated the icons to the orange palette

**Comment by janniks** [comment]
Cool thanks!

The landing page update looks great 👍🏻 very clean

**Comment by janniks** [comment]
This PR now also removes the survey link. https://github.com/hirosystems/docs/issues/167

@sabbyanandan what's our current review&release flow for the docs site?

**Comment by sabbyanandan** [comment]
> @sabbyanandan what's our current review&release flow for the docs site?

There's a major left-nav redo in progress, which is targeted to merge by Q3 (end of this week). And I would suggest waiting this out until we get those changes promoted to production first. 

As for reviews, @trichards-hiro and @LakshmiLavanyaKasturi are the primary owners, so they should be able to give feedback.

**Comment by LakshmiLavanyaKasturi** [comment]
Changes LGTM! As Sabby mentioned we would want to wait until the left nav changes are pushed and re-iterate on the "References" dropdown alone to make sure there is no duplication.

**Comment by LakshmiLavanyaKasturi** [comment]
@janniks: Now that we have our left-nav changes merged, can you resolve the conflicts and merge this PR?

**Comment by LakshmiLavanyaKasturi** [comment]
@janniks: Just wondering if we have any dependency to merge this PR? If not, can we do it?

**Comment by janniks** [comment]
Yes I'll get on that

**Comment by janniks** [comment]
@LakshmiLavanyaKasturi I think this can be re-reviewed now

**Comment by LakshmiLavanyaKasturi** [comment]
> @LakshmiLavanyaKasturi I think this can be re-reviewed now _(if we want to go in this direction)_

Will look into it. Thanks, @janniks

**Comment by trichards-hiro** [comment]
@janniks I also really like these changes. The flow looks better. The only question I have about the placement of the Stacks API content under References is about whether you think this will impact our SEO.
